server/api/offline_batch_results.py

```diff
@@ -185,6 +185,89 @@ def record_offline_batch_results(
     return jsonify(status="ok")
 
 
+@api.route(
```

Comment:
idea for forcing users to update: keep the existing endpoint above but make it just return an error message that says "Arlo has been updated. Please refresh your browser for the latest version."

Reply:
I expect some counties will need to parallelize 4-5x, constantly entering data. We could literally see livelock. So no, I think we have to allow for truly parallel addition.